ddl/delete_range.go

```diff
@@ -243,6 +243,25 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 		startKey = tablecodec.EncodeTablePrefix(tableID)
 		endKey := tablecodec.EncodeTablePrefix(tableID + 1)
 		return doInsert(s, job.ID, tableID, startKey, endKey, now)
+	case model.ActionCreateTable:
```

Comment:
Can we change the job Args to same with `drop table`? Then we can reuse the `drop table` code here`.

Reply:
Or create a function to remove redundant code in here and `drop table`.

Reply:
Thanks for the tips, let me refine the code